README.md

```diff
@@ -350,6 +350,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1600+ peo
   * [Svix](https://www.svix.com/) - Webhooks as a Service. Send up to 50,000 messages/month for free.
   * [TemplateTo](https://templateto.com) - Auto-Generate PDF/TXT documents from reusable templates with our drop-and-drop editor and simple API. The free plan comes with 450 PDFs/month and three templates.
   * [TinyMCE](https://www.tiny.cloud) - rich text editing API. Core features are free for unlimited usage.
+  * [Treblle](https://www.treblle.com) - Treblle helps teams build, ship, and govern APIs. With advanced API log aggregation, observability, docs, and debugging, it enables API-first companies to innovate and drive progress efficiently.
```

Comment:
Replace `, it enables API-first companies to innovate and drive progress efficiently.` with details of what's offered for free please.  Thanks. Better focus on details and not marketing speak that isn't that useful.